spiketoolkit/comparison/basecomparison.py

```diff
@@ -0,0 +1,134 @@
+import spikeextractors as se
+
```

Comment:
add `import numpy as np` for the moment to plot confusion matrix

Reply:
OK